front_end/content.py

```diff
@@ -76,10 +76,9 @@ def create_database_tables(self):
                           name text,
                           given_name text,
                           family_name text,
```

Comment:
The following changes are just reverting the function 'create_database_tables' to how it was before; in past commits I didn't realize that I shouldn't touch this function and in changing the way migration scripts run just now I stumbled into a few problems. Everything works again though!